build.gradle

```diff
@@ -148,6 +148,7 @@ dependencies {
     compile group: 'mrj', name: 'MRJ141Stubs', version:'0'
     compile group: 'net.sf.squirrel-sql.thirdparty-non-maven', name: 'skinlf', version:'6.7'
     compile group: 'org.apache.xmlgraphics', name: 'fop', version:'2.1'
+    compile group: 'org.apache.pdfbox', name: 'pdfbox', version:'1.8.9'
```

Comment:
pdfbox, or rather fontbox which is a part of pdfbox, is needed by fop to embed custom fonts.